### PR TITLE
#39 Justifications API

### PR DESCRIPTION
**Pull request by NFSS10**
Related to issue: [#39](https://github.com/ripe-tech/ripe-pulse/issues/39)
Description: Can now get the "justifications" list. They can optionally be filtered by "context".

### CODE REVIEW

File: src/js/api/justification.js [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+}
+
+/**
+ * Gets the justifications list. Can be optionally filtered by context.
+ *
+ * @param {Object} options An object of options to configure the query and
+ * its results, such as:
+ * - 'filters[]' - List of filters that the query will use to, operators such as
+ * ('in', 'not_in', 'like', 'likei', 'llike', 'llikei', 'rlike', 'rlikei', 'contains'),
+ * (eg: 'context:eq:42') would filter by the 'context' that equals to '42'.
```
Comment (gcandal) [inline src/js/api/justification.js]: Don't use context as an example, it gets a bit confusing given that it is a parameter.

File: src/js/api/justification.js [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+ * - 'sort' - List of arguments to sort the results by and which direction
+ * to sort them in (eg: 'id:ascending') would sort by the id attribute in ascending order,
+ * while (eg: 'id:descending')] would do it in descending order.
+ * - 'skip' - The number of the first record to retrieve from the results.
+ * - 'limit' - The number of results to retrieve.
+ * @param {Function} callback Function with the result of the request.
+ * @returns {XMLHttpRequest} The XMLHttpRequest instance of the API request.
+ */
+ripe.Ripe.prototype.getJustifications = function(context = null, options, callback) {
```
Comment (gcandal) [inline src/js/api/justification.js]: Having an optional with a default (context) followed by other parameters without it is a bit weird. Is that the way we're doing for other API methods?

**Comment by gcandal** [comment]
Also, the build seems to be failing, so please take a look at it.

**Comment by gcandal** [comment]
@NFSS10 assign me back once the proper fix is pushed

**Comment by gcandal** [comment]
What happened to the `context`?

**Comment by NFSS10** [comment]
> What happened to the `context`?

Added new changed after we cleared the issue.

**Pull request merged**